Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -586,6 +586,31 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    // https://www.hackingwithswift.com/articles/141/8-useful-swift-extensions
```

Comment:
@zmfrew could you please move the source comment to be inside the function

Reply:
I think it should be in `See also:` in the method documentation